### PR TITLE
LPS-159824 | Add SiteLevelWalkthrough#CanEnableSkippable testcase

### PR DESCRIPTION
**Pull request by mateusralv**
Background:
Create automated test SiteLevelWalkthrough#CanEnableSkippable

Test Plan:
**Given** walkthrough enabled site-level
**And Given** skippable is set to true
**When** on step 2
**And When** enable "Do not show me this again"
**And When** navigate to next step
**And When** click ok to finish the steps
**Then** popover mode is not present

JIRA Link:
https://issues.liferay.com/browse/LPS-159824

### CODE REVIEW

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by john-co** [comment]
I feel like this test has already been covered by this unit test: https://github.com/liferay-frontend/liferay-portal/blob/90e2c79010cf33c6369268f533cc3e65c728402b/modules/apps/frontend-js/frontend-js-walkthrough-web/test/js/Walkthrough.test.js#L257-L275

Could you please confirm @diegonvs and @mateusralv ?

**Comment by mateusralv** [comment]
Hi @john-co, yes but we're relying on Browser's localStorage. Diego explained a little [here](https://docs.google.com/spreadsheets/d/1c--LZnVJAPR2JbUZoAizjNXNbWtX54KKAVk465sizjo/edit?disco=AAAAf6XtGLs) about keeping this test.

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/macros/Walkthrough.macro [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+				widgetName = "JS Walkthrough Sample");
+		}
+
+		task ("When instantiate the walkthrough") {
+			Navigator.openSitePage(
+				pageName = "Walkthrough Page",
+				siteName = "Site Name");
+
+			Walkthrough.enablePopoverMode();
+		}
```
Comment (john-co) [inline modules/apps/frontend-js/frontend-js-test/src/testFunctional/macros/Walkthrough.macro:27]: ooh, interesting idea to add task descriptions within the macro.

**Review by john-co — APPROVED** [review]
Looks good overall, I made a refactor on some macros here that were merged recently: https://github.com/liferay-frontend/liferay-portal/pull/2721/commits

Also, I want to start following a naming pattern for our testcases: https://docs.google.com/document/d/1oRYHQ2btff7W0iX1JU3oBPFCJeqBo3emajKp-kujKUs/edit#heading=h.y4s0uxrhrbwt So following the pattern, let's change the testcase file name to `WalkthroughSiteLevel.testcase`

Please study and incorporate into the CanEnableSkippable testcase. After that, feel free to forward.

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:forward

**Comment by mateusralv** [comment]
ci:stop

**Comment by mateusralv** [comment]
ci:forward

**Comment by mateusralv** [comment]
ci:reevaluate:1377401_5265

**Comment by mateusralv** [comment]
ci:reevaluate:1377401_5265

**Comment by mateusralv** [comment]
ci:reevaluate:1377401_5265

**Comment by mateusralv** [comment]
ci:reavaliar:1377401_5265

**Pull request closed without merging**